cluster/cluster.sh

```diff
@@ -49,5 +49,12 @@ function cluster::path() {
 }
 
 function cluster::kubeconfig() {
-    echo -n ${CLUSTER_PATH}/_ci-configs/${KUBEVIRT_PROVIDER}/.kubeconfig
+
+    if [ ${KUBEVIRT_PROVIDER} != "external" ]; then
+      echo -n ${CLUSTER_PATH}/_ci-configs/${KUBEVIRT_PROVIDER}/.kubeconfig
+    fi
+    if [ -n ${KUBECONFIG} ]; then
+      echo -n ${KUBECONFIG}; else
```

Comment:
this if / else syntax confuses me, but I'm awful at bash, and thus get easily confused.

Reply:
I am more awful than you that's for sure. Is there a better way you can think of?

Reply:
```bash
if [ -n ${KUBECONFIG} ]; then
      echo -n ${KUBECONFIG}
else
    ...
fi
```
:)

Reply:
[Done](https://github.com/kubevirt/cluster-network-addons-operator/compare/060c6dcf967e8120917b3b52828147008d7234e9..fd1feb96a75c8b97e7f11975c86f1d7218f840b5)